Libs/Visualization/VTK/Widgets/ctkVTKScalarsToColorsComboBox.cpp

```diff
@@ -85,6 +85,7 @@ ctkVTKScalarsToColorsComboBox::ctkVTKScalarsToColorsComboBox(QWidget* _parent)
 // --------------------------------------------------------------------------
 ctkVTKScalarsToColorsComboBox::~ctkVTKScalarsToColorsComboBox()
 {
+  QComboBox::clear();
```

Comment:
which leak is this addressing ?

Reply:
This triggers the method "onRowsAboutToBeRemoved" which deletes the vtkScalarsToColors in the combobox.

Reply:
I see.

Relying on polymorphism (aka virtual function) from within the destructor is usually not recommended. See https://stackoverflow.com/questions/12092933/calling-virtual-function-from-destructor#12093250

I suggest to directly call `onRowsAboutToBeRemoved`

Reply:
An other approach would be directly manage `vtkSmartPointer` but i don't know if this possible here